### ISSUE TITLE
 Godot 4 project convertor failing to load a music file

### ISSUE THREAD

**Issue by dardasaba6** [godotengine/godot#73393]
### Godot version

4.0 RC1

### System information

Linux Mint 20.3 Cinnamon

### Issue description

Tried converting a 3.5 game to 4.0 RC1. The conversion popup menu closed after a few seconds, and trying to edit the game crashes the entire engine with the following errors, which seem to be related to one music file not loading:
![errors](https://user-images.githubusercontent.com/125468288/219127087-f5a78035-4e4a-4625-aadb-e0cde6d5ac3d.png)

Not sure what's causing the issue, because when I created a new project with only that file and converted it I didn't get these errors, but consistently get them when trying to convert that game.

### Steps to reproduce

- Download the reproduction project below
- Import in Godot 4.0, and click "convert full project"
- If the bug happens, the engine will crash whenever trying to edit or run the converted project

### Minimal reproduction project

[Music Convertor Crash Demo.zip](https://github.com/godotengine/godot/files/10746591/Music.Convertor.Crash.Demo.zip)

**Comment by gotnospirit** [comment]
4.0 RC1 on Windows 10:

While many errors were prompted in console and the main scene wasn't loaded in the editor, I didn't had crash and was able to run the converted project.

Now, If I remove the ".import" folder and the files with ".import" extension before the "import & convert" process, most of the errors are gone and the main scene is successfully loaded.

For example, the ogg vorbis importer's name changed between 3.x & 4.x but the converter don't treat these "import" files :
https://github.com/godotengine/godot/blob/3.x/modules/stb_vorbis/resource_importer_ogg_vorbis.cpp#L37-L39
https://github.com/godotengine/godot/blob/master/modules/vorbis/resource_importer_ogg_vorbis.cpp#L44-L46

It may be a good idea to have the project converter removes the ".import" files/folder

**Comment by YuriSizov** [comment]
You should not remove .import files, they hold information for your import configuration. By removing those files you reset them to defaults, even those which worked perfectly fine before. The folder you can remove, it's transient and is not used in Godot 4 (we now have .godot instead).

**Comment by gotnospirit** [comment]
Ok. Just tried with only removing the import folder and had the same problems as initial report, so it seems that project converter should handle convertion for ".import" files too

**Comment by MJacred** [comment]
The errors changed in godot 4.0.2: the music file does not create any issues anymore

my error log on Ubuntu 20.04, it's all `.ttf` (font) related:

```sh
  editor/export/editor_export.cpp:227 - Condition "!preset.is_valid()" is true. Continuing.
--- Debug adapter server started ---
--- GDScript language server started ---
  editor/editor_file_system.cpp:2267 - Condition "!importer.is_valid()" is true. Continuing.
  editor/editor_file_system.cpp:2267 - Condition "!importer.is_valid()" is true. Continuing.
  No loader found for resource: res://m6x11.ttf.
  scene/resources/resource_format_text.cpp:174 - res://MainMenu.tscn:10 - Parse Error: [ext_resource] referenced non-loaded resource at: res://m6x11.ttf
  scene/resources/resource_format_text.cpp:621 - res://MainMenu.tscn:10 - Parse Error: [ext_resource] referenced non-loaded resource at: res://m6x11.ttf
  Failed loading resource: res://MainMenu.tscn. Make sure resources have been imported by opening the project in the editor at least once.
  editor/editor_data.cpp:574 - Index p_idx = 1 is out of bounds (edited_scene.size() = 1).
```

**Comment by akien-mga** [comment]
Fixed by #74580.
